checks/check-nginx-errors/check-nginx-errors.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/AirHelp/sensu-plugins-go/elasticsearch"
+
+	"github.com/AirHelp/sensu-plugins-go/sensu"
+	"github.com/AirHelp/sensu-plugins-go/utils"
+	"github.com/olivere/elastic/v7"
+)
+
+var options = &struct {
+	url          string
+	indexPattern string
+	role         string
+
+	statuses statuses
+	minutes  int
+
+	showVersion bool
+	debug       bool
+}{}
+
+var esClient elasticsearch.EsClientIface
+
+type statusConfig struct {
+	code, warning, critical int
+}
+
+type statuses []statusConfig
+
+func (i *statuses) String() string {
+	s, _ := json.Marshal(i)
+	return string(s)
+}
+
+func (i *statuses) CodesAsInterface() []interface{} {
+	res := []interface{}{}
+
+	for _, j := range *i {
+		res = append(res, j.code)
+	}
+
+	return res
+}
+
+func (i *statuses) Set(value string) error {
+	statuses := strings.Split(value, ";")
```

Comment:
maybe it will be less error prone if we cut out the white spaces 🤔 
```suggestion
	statuses := strings.Split(strings.ReplaceAll(value, " ", ""), ";")
```